odl/ufunc_ops/ufunc_ops.py

```diff
@@ -199,6 +199,7 @@ def __repr__(self):
         """Return ``repr(self)``."""
         return '{}({!r})'.format(name, self.domain)
 
+    """
```

Comment:
That's just temporary since it made importing ODL fail as long as the interface wasn't working properly.